src/chromium/crBrowser.ts

```diff
@@ -301,6 +302,7 @@ export class CRBrowserContext extends BrowserContextBase {
     }
     if (this._options.permissions)
       promises.push(this.grantPermissions(this._options.permissions));
+    promises.push(debugSupport.installConsoleHelpers(this));
```

Comment:
Can we have this in a browser-neutral manner?

Reply:
Wrapped in `BrowserContextBase._initialize`.